modules/oc-mirror-command-reference.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-command-reference_{context}"]
+= Command reference for oc-mirror
+
+The following tables describe the `oc mirror` subcommands and flags:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
+|`help`
+|Show help about any subcommand.
+
+|`init`
+|Output an initial image set configuration template.
+
+|`list`
+|List available platform and Operator content and their version.
+
+|`version`
+|Output the oc-mirror version.
+
+|===
+
+.oc mirror flags
+[cols="1,2",options="header"]
+|===
+|Flag
+|Description
+
+|`-c`, `--config` `<string>`
+|Specify the path to an image set configuration file.
+
+|`--continue-on-error`
+|If an error occurs, continue and attempt to mirror as much as possible.
+
+|`--dest-skip-tls`
+|Disable TLS validation for the target registry.
+
+|`--dest-use-http`
+|Use plain HTTP for the target registry.
+
+|`--dry-run`
+|Print actions without mirroring images. Generates `mapping.txt` and `pruning-plan.json` files. This flag is experimental and only works for mirroring to disk.
```

Comment:
`--dry-run` not only works for mirroring to disk, also for mirror2mirror .

Reply:
Thanks, removed that whole sentence since Jenn said that it's not experimental anymore either